.travis.yml

```diff
@@ -3,6 +3,7 @@ rvm:
   - 1.9.3
 env: facebook_public_key=test facebook_private_key=test google_public_key=test google_private_key=test
 script:
+  - RAILS_ENV=test bundle exec rake db:migrate --trace
```

Comment:
That forum says to run `rake db:test:prepare` instead of `RAILS_ENV=test bundle exec rake db:migrate`. Try with the former command, because it's much cleaner.